.eslintrc.yml

```diff
@@ -48,14 +48,13 @@ settings:
     version: 16.3.2
 
 overrides:
-  - files: '*.{md,mdx}'
-    extends:
-      - plugin:mdx/recommended
-      - plugin:mdx/overrides
-    rules:
-      import/no-extraneous-dependencies: 0
-      prettier/prettier: error
-
+  # - files: '*.{md,mdx}'
+  #   extends:
+  #     - plugin:mdx/recommended
+  #     - plugin:mdx/overrides
+  #   rules:
+  #     import/no-extraneous-dependencies: 0
+  #     prettier/prettier: error
```

Comment:
why did a bunch of eslint rules need to be disabled?

Reply:
/cc @JounQin

This now uses the MDX@2 parsing, which conflicts with eslint-mdx.
I want to add `eslint-mdx` again when it supports the things mdx@next does and will once v2 is released.

Reply:
Should `remark-lint` be added back for now in this case?

Reply:
Good idea!

Also Joun: there’s still a lot to do here to port xdm into mdx-js/mdx and write a lot of documentation.
I’m hoping to have a release (candidate) of v2 around in 4 weeks.
Do you have questions or need other information that I can help with for `eslint-mdx`?

Reply:
The problem is that I don't have enough time to work on `next` version for these days. 😂